wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/DeviceDetailsViewActionsHandler.swift

```diff
@@ -59,15 +59,9 @@ final class DeviceDetailsViewActionsHandler: DeviceDetailsViewActions, Observabl
     }
 
     @MainActor
-    func updateCertificate() async throws -> E2eIdentityCertificate? {
-        try await enrollClient()
-    }
-
-    @MainActor
-    func enrollClient() async throws -> E2eIdentityCertificate? {
+    func enrollClient() async throws -> String? {
```

Comment:
question: what's the return type? maybe add what it returns as doc. + does it have to be optional